pkg/resource/claim_reconciler.go

```diff
@@ -370,6 +372,21 @@ func (r *ClaimReconciler) Reconcile(req reconcile.Request) (reconcile.Result, er
 			return reconcile.Result{RequeueAfter: aShortWait}, errors.Wrap(r.client.Status().Update(ctx, claim), errUpdateClaimStatus)
 		}
 
+		if !IsConditionTrue(claim.GetCondition(v1alpha1.TypeReferencesResolved)) {
```

Comment:
Just wanted to make a note here because I think we discussed removing this since we do it in the managed reconciler.